#GNSS Constellations configured
GPS: 1
GLONAS: 0
BEIDOU: 1
GALILEO: 1

#moving or stationary base
base_type: "stationary"  

#if stationary the survey time and survey accuracy minimums
Surveytime: 30
Surveyacc: 5000000

#GNSS message output rate
message_rate: 5

#These values are just for the base.  
#They need to be changed in the launch file for the rover
rover_quantity: 1    #number of rovers





